actions.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Why does your code look pretty much similar to `https://github.com/MariaChizhova/SE_2021/pull/2/files#diff-3de198f19909fc784ca274482454308ace5fdcea3049209a38b5c002333acfcfR1`?

Reply:
Actually, we did not have any experience in writing such scripts before, so we had to google everything. So, frankly speaking, the majority of this code really was taken from some side resources.
I can try to find links/files, from which we took the code and combined it to create ours, if you need to have a look at them.